### PR TITLE
fix(curriculum): 🧹 minor typo corrections detected by web crawler

### PR DESCRIPTION
**Pull request by spencerlepine**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Found typos in this repository and made suggestions to correct.

> __Note:__ this was detected by the _open source web crawler project 🕷_ - [spencerlepine/open-source-crawler](https://github.com/spencerlepine/open-source-crawler)

### CODE REVIEW

**Review by raisedadead — CHANGES_REQUESTED** [review]
I assume these changes were not tested manually for validity because of your automation.

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-122-efficient-exponentiation.md [hunk 1]
```diff
@@ -39,23 +39,23 @@ For $1 ≤ k ≤ 200$, find $\sum{m(k)}$.
 
 # --hints--
 
-`efficientExponentation()` should return `1582`.
+`efficientExponentiation()` should return `1582`.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-122-efficient-exponentiation.md:42]: The changes in this file should be reveresed

Comment (naomi-lgbt) [reply curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-122-efficient-exponentiation.md:42]: I assume we did - AFAICT, Euler doesn't even care about functions.

**Comment by raisedadead** [comment]
Thank you for opening this pull request.

We’ve reviewed your pull request and decided not to merge it; However, We would welcome future pull requests from you. 

You are welcome to [read these guidelines](https://contribute.freecodecamp.org) to learn more about contributing to this project. 

Happy contributing.

**Pull request closed without merging**